index.html

```diff
@@ -53,8 +53,9 @@
         <div class="row">
           <div class="col-sm-8 col-md-7 py-4">
             <h4 class="text-white">About</h4>
-            <p class="text-white">OpenBSD and NetBSD will resize the root partition during the first boot.</p>
             <p class="text-white">All the images come with a single root partition, and use UFS/FFS filesystem.</p>
+            <p class="text-white">NetBSD will resize the root partition during the first boot.</p>
+            <p class="text-white">OpenBSD is provided with a script (<code>/root/bin/create_partitions.sh</code>) to adapt the partition layout after the first boot. The games and X server sets are not installed to keep the image minimal.</p>
```

Comment:
Could you also update line 68-69 (the "The scripts used to build the images are listed below:" block), and point on your repository?

Reply:
I think you already did it in this commit: 562276f46eb48b1067aae5217795c29a11facab9

Reply:
Oops!